docker-compose.yml

```diff
@@ -9,15 +9,16 @@ services:
     environment:
       ethereum: http://ganache:8545
       test_mnemonic: myth like bonus scare over problem client lizard pioneer submit female collect
-      node_http: http://graph-node:8000/subgraphs/name/daostack/graphql
+      node_http: http://graph-node:8000/subgraphs/name/daostack
       node_ws: http://graph-node:8001/subgraphs/name/daostack
       node_rpc: http://graph-node:8020
       ipfs_host: ipfs
       ipfs_port: 5001
       WAIT_HOSTS: ipfs:5001, graph-node:8020, ganache:8545
 
   graph-node:
-    image: graphprotocol/graph-node:v0.5.0
+    image: graphprotocol/graph-node:latest@sha256:a11adda030fcb4ac42d0c31c58cffaacadfda333fecfb1e361d9ec5a72ba1026
```

Comment:
is this guaranty a specific version?

Reply:
yes exactly, I think this is the most important part of this PR. We were  basically pulling a new image every time they update theirs, which makes our enviroment unreliable.